test/verifiedTests/analysis/testMultiSpeciesModelling/SRP065497_taxonomy_abundances_v3.0.csv

```diff
@@ -0,0 +1,866 @@
R2761038,SRR2761039,SRR2761040,SRR2761041,SRR2761042,SRR2761043,SRR2761044,SRR2761045,SRR2761046,SRR2761047,SRR2761048,SRR2761049,SRR2761050,SRR2761051,SRR2761052,SRR2761053,SRR2761054,SRR2761055,SRR2761056,SRR2761057,SRR2761058,SRR2761059,SRR2761060,SRR2761061,SRR2761062,SRR2761063,SRR2761064,SRR2761065,SRR2761066,SRR2761067,SRR2761068,SRR2761069,SRR2761070,SRR2761071,SRR2761072,SRR2761073,SRR2761074,SRR2761075,SRR2761076,SRR2761077,SRR2761078,SRR2761079,SRR2761080,SRR2761081,SRR2761082,SRR2761083,SRR2761084,SRR2761085,SRR2761086,SRR2761087,SRR2761088,SRR2761089,SRR2761090,SRR2761091,SRR2761092,SRR2761093,SRR2761094,SRR2761095,SRR2761096,SRR2761179,SRR2761180,SRR2761181,SRR2761182,SRR2761183,SRR2761184,SRR2761185,SRR2761186,SRR2761187,SRR2761188,SRR2761189,SRR2761190,SRR2761191,SRR2761192,SRR2761193,SRR2761194,SRR2761195,SRR2761196,SRR2761197,SRR2761198,SRR2761199,SRR2761200,SRR2761201,SRR2761202,SRR2761203,SRR2761204,SRR2761205,SRR2761206,SRR2761207,SRR2761208,SRR2761209,SRR2761210,SRR2761211,SRR2761212,SRR2761213,SRR2761214,SRR2761215,SRR2761217,SRR2761218,SRR2761219,SRR2761220,SRR2761221,SRR2761222,SRR2761223,SRR2761224,SRR2761226,SRR2761227,SRR2761228,SRR2761229,SRR2761230,SRR2761231,SRR2761232,SRR2761234,SRR2761235,SRR2761236,SRR2761237,SRR2761238,SRR2761239,SRR2761240,SRR2761241,SRR2761242,SRR2761243,SRR2761244,SRR2761245,SRR2761246,SRR2761247,SRR2761248,SRR2761249,SRR2761250,SRR2761251,SRR2761252,SRR2761253,SRR2761254,SRR2761255,SRR2761256,SRR2761257,SRR2761258,SRR2761259,SRR2761260,SRR2761261,SRR2761262,SRR2761263,SRR2761264,SRR2761265,SRR2761266,SRR2761267,SRR2761268,SRR2761269,SRR2761270,SRR2761271,SRR2761272,SRR2761273,SRR2761274,SRR2761275,SRR2761276,SRR2761278,SRR2761279,SRR2761280,SRR2761281,SRR2761282,SRR2761283,SRR2761284,SRR2761285,SRR2761286,SRR2761287,SRR2761288,SRR2761289,SRR2761290,SRR2761291,SRR2761292,SRR2761293,SRR2761294,SRR2761295,SRR2761296,SRR2761297,SRR2761298,SRR2761299,SRR2761300,SRR2761301,SRR2761302,SRR2761303,SRR2761304,SRR2761305,SRR2761306,SRR2761307,SRR2761308,SRR2761309,SRR2761310,SRR2761311,SRR2761312,SRR2761314,SRR2761315,SRR2761316,SRR2761317,SRR2761318,SRR2761319,SRR2761320,SRR2761321,SRR2761322,SRR2761338,SRR2761340,SRR2761341,SRR2761342,SRR2761343,SRR2761344,SRR2761345,SRR2761346,SRR2761347,SRR2761348,SRR2761349,SRR2761350,SRR2761351,SRR2761352,SRR2761353,SRR2761354,SRR2761355,SRR2761356,SRR2761357,SRR2761358,SRR2761359,SRR2761360,SRR2761361,SRR2761362,SRR2761363,SRR2761365,SRR2761366,SRR2761368,SRR2761369,SRR2761370,SRR2761371,SRR2761372,SRR2761373,SRR2761374,SRR2761375,SRR2761376,SRR2761377,SRR2761378,SRR2761379,SRR2761380,SRR2761381,SRR2761382,SRR2761383,SRR2761384,SRR2761385,SRR2761386,SRR2761387,SRR2761388,SRR2761389,SRR2761390,SRR2761391,SRR2761392,SRR2761393,SRR2761394,SRR2761395,SRR2761396,SRR2761397,SRR2761399,SRR2761400,SRR2761401,SRR2761402,SRR2761403,SRR2761404,SRR2761405,SRR2761406,SRR2761407,SRR2761408,SRR2761409,SRR2761410,SRR2761411,SRR2761412,SRR2761413,SRR2761414,SRR2761415,SRR2761416
```

Comment:
May you rename `SRP065497_taxonomy_abundances_v3.0.csv` according to the guidelines? Or where can we download this file from?

Reply:
What are the guidelines? The file comes from here: https://www.ebi.ac.uk/metagenomics/api/v1/studies/MGYS00001248/pipelines/3.0/file/SRP065497_taxonomy_abundances_v3.0.tsv. The file needs to be in csv format (not tsv), so we can't just download it.

Reply:
Best would be to download the .csv file using system commands. Why do you have to have the file in .tsv? We can also create the .tsv from the .csv using `sed`.

Reply:
It's the other way around, the file is provided in tsv but needs to be csv. The readtable function doesn't work on tsv files.

Reply:
Ah ok - but the same applies.

Reply:
`readtable` works with tsv files. you would just have to set `'delimiter','tab'` in the arguments.

Reply:
How does converting the file work?

Reply:
@tpfau No, I get
Error using readtable (line 216)
The file extension '.tsv' is not recognized.

Reply:
What you could try is to determine the text file format (maybe an additional general function for this would be an idea), and thus allow both csv and tsv files (or potentially even try semi-colon).
This might help quite some people exporting from excel.

Reply:
Argh..
So it would read tab-separated files but only if they come in as a .csv...
What should work is:
readtable('FileName.tsv','FileType','text','delimiter','tab');

Reply:
@tpfau That works, thanks! +1:  I will modify the function accordingly.

Reply:
I just noticed, it should probably also work, if you don't define the delimiter, could you give that a try, as it would make it more flexible.

Reply:
@laurentheirendt argh....suddenly, the download function doesn't work anymore.
system('curl -O https://www.ebi.ac.uk/metagenomics/api/v1/studies/MGYS00001248/pipelines/3.0/file/SRP065497_taxonomy_abundances_v3.0.tsv')
'curl' is not recognized as an internal or external command, 
operable program or batch file. 
curl.exe is in my path so I am not sure what is going on.

Reply:
May you check if everything is OK with the function now?

Reply:
What do I have to check?

Reply:
If the test function runs through. If it is fine, can you accept the pull request?

Reply:
Well, the tests don't pass yet as you can see.

Reply:
What exactly is not passing?